core-metadata/pom.xml

```diff
@@ -24,6 +24,18 @@
     <packaging>jar</packaging>
     <name>Apache Kylin - Core Metadata</name>
     <description>Apache Kylin - Core Metadata</description>
+    <build>
```

Comment:
Do you mean this commit only works in JDK8+?

Reply:
I use stream api in my code. And I found UT on master branch with JDK 1.7(deprecated): Kylin-Master-JDK-1.7  in kylin docs.So I add this config.